### DISCUSSION TITLE
Revamped Visual Studio Resource Explorer breaks ResXResourceManager

### DISCUSSION THREAD

**Discussion by cchiarelli** [dotnet/ResXResourceManager#659] [Q&A] [answered]
Looks like Microsoft updated the Resource Editor (circa June 2024)  ... they now call it Resource Explorer. Resx will no longer load in the resource  files. Anyone having the same issues, any fixes?

**Comment by tom-englert** [comment]
![image](https://github.com/user-attachments/assets/2c10b87d-4695-4229-976c-cff95de8be0a)

Open in ResX Manager still works fine, so what exactly is your problem?

**Reply by cchiarelli** [reply]
Creating a simple Localization demo in Blazor .NET 8  with US english and French Canadian options. See image below. The default Resource is US. If I try renaming it to Resource.en-US.resx I no longer see it in Resource Explorer. I change it back to just Resource and all my definitions reappear. Below you see the error message after I add a new Resource file (Resource.fr-CA.resx) and then try to click on it to add my defintions as I used to do prior to this new Version of the Resource Explorer.

![localization](https://github.com/user-attachments/assets/0f8720d5-3c89-4af1-9995-33f692233846)

**Reply by cchiarelli** [reply]
I've have reverted back to the legacy editor and set it as the default for now.

**Comment by tom-englert** [comment, marked as answer]
@cchiarelli This is the discussion forum of ResXResourceManager. 
If you have problems with the built in editors of Visual Studio, please discuss them in the Visual Studio forum.

**Reply by cchiarelli** [reply]
Sorry about that Tom. Newbie on the forum
